### ISSUE TITLE
scan_bucket_urls: Max retries exceeded with url

### ISSUE THREAD

**Issue by vladgolubev** [kromtech/s3-inspector#7]
Just scanned my buckets, all are ok, but failed on the last one `www.gemshelf.com`

Looks like regex issue? Here is the stacktrace

```sh
Traceback (most recent call last):
  File "s3inspector.py", line 109, in <module>
    urls = scan_bucket_urls(bucket.name)
  File "s3inspector.py", line 47, in scan_bucket_urls
    content = requests.get(url).text
  File "/usr/local/lib/python2.7/site-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/adapters.py", line 506, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='www.gemshelf.com.s3.amazonaws.com', port=443): Max retries exceeded with url: / (Caused by SSLError(CertificateError("hostname 'www.gemshelf.com.s3.amazonaws.com' doesn't match either of '*.s3.amazonaws.com', 's3.amazonaws.com'",),))
```

**Comment by Volfar** [comment]
Hi, @vladgolubev.
Thanks for participating in our project.
The bug you described is not in our tool, but rather in how Amazon incorrectly issues SSL certificates to buckets with dots in their names.
In this case, the certificate was issued to ```*.s3.amazonaws.com``` which doesn't equal to ```*.*.*.s3.amazonaws.com```, which would be needed for https://www.gemshelf.com.s3.amazonaws.com
I added simple error handling for such case and it is now fixed in e65af2a4f2df1ca0d95e86dc2e5773b91118c799.
